src/Threebox.js

```diff
@@ -1094,21 +1094,24 @@ Threebox.prototype = {
 
 	//[jscastro] method to fully dispose the resources, watch out is you call this without navigating to other page
 	dispose: async function () {
+		this.disposed = true;
 
-		console.log(this.memory());
+		if (this.options.logMemory) {
+			console.log(this.memory());
+		}
 		//console.log(window.performance.memory);
 
 		return new Promise((resolve) => {
 			resolve(
 				this.clear(null, true).then((resolve) => {
-					this.map.remove();
-					this.map = {};
```

Comment:
I think we are already remove the map ourselves already, removing it twice seems to throw errors.